packages/protocol/test/governance/election.ts

```diff
@@ -901,23 +900,6 @@ contract('Election', (accounts: string[]) => {
       })
     })
 
-    describe('when different random values are provided', () => {
```

Comment:
Now that we're not using this, can we remove all references to `random` in this test?

Reply:
Even `registry.setAddressFor(CeloContractName.Random, random.address)` ? I'll look into in separate PR.